doc/design/cluster_train/data_dispatch.md

```diff
@@ -107,13 +109,14 @@ trainer.train(batch_reader, ...)
 
 使用下面命令，可以把本地的数据上传到存储集群中。
 
-```bash
-paddle cp filenames pfs://home/folder/
+```bash  
+paddle pfs cp filenames /pfs/folder/
```

Comment:
It is pretty common that a user might need to access more than one clusters. For example, I need to access the Key State Lab's cluster and the 3 or 4-node cluster @typhoonzero set up for development and demonstrations.